data-rights-protocol.md

```diff
@@ -190,27 +195,30 @@ This table shows valid states for Data Rights Requests, along with the criteria
 | denied      | suspected_fraud        | CB or PIP believes this request was made fraudulently               | processing_details                           | x      |
 | denied      | insuf_verification     | the [in_progress, need_user_verification] stage failed or timed out | processing_details                           | x      |
 | denied      | no_match               | CB could not match user identity to data subject                    | processing_details                           | x      |
-| denied      | claim_not_covered      | user requesting data not covered under legal bases[XXX]             | processing_details                           | x      |
-| denied      | outside_jurisdiction   | user requesting data under bases they are not covered by[XXX]       | processing_details                           | x      |
+| denied      | claim_not_covered      | user requesting data not covered under legal bases[2]             | processing_details                           | x      |
+| denied      | outside_jurisdiction   | user requesting data under bases they are not covered by[2]       | processing_details                           | x      |
 | denied      | too_many_requests      | user has submitted more requests than the CB is legally obliged to process | details?
 | denied      | other                  | some other unspecified failure state reached                        | processing_details                           | x      |
 | expired     |                        | the time is currently after the `expires_at` in the request.        |                                              | x      |
 
-[XXX] in the case of claim_not_covered, this may be about asking for categories of data which Covered Businesses are not required to present to the User. in the case of outside_jurisdiction, this may be because the business is not honoring CCPA requests for non-California residents and there is no other basis on which to honor the request. [#28](https://github.com/consumer-reports-digital-lab/data-rights-protocol/issues/28) for discussion on `too_many_requests`
+[2]: in the case of claim_not_covered, this may be about asking for categories of data which Covered Businesses are not required to present to the User. in the case of outside_jurisdiction, this may be because the business is not honoring CCPA requests for non-California residents and there is no other basis on which to honor the request. [#28](https://github.com/consumer-reports-digital-lab/data-rights-protocol/issues/28) for discussion on `too_many_requests`
```

Comment:
```suggestion
[2]: In the case of claim_not_covered, this may be about asking for categories of data which Covered Businesses are not required to present to the User. In the case of outside_jurisdiction, this may be because the business is not honoring CCPA requests for non-California residents and there is no other basis on which to honor the request. See [#28](https://github.com/consumer-reports-digital-lab/data-rights-protocol/issues/28) for discussion on `too_many_requests`.
```